ext/autocomplete/style.css

```diff
@@ -0,0 +1,8 @@
+#Navigationleft .blockbody { overflow: visible; }
+
+.tagit { background: white !important; border: 1px solid grey !important; cursor: text; }
+.tagit-choice { cursor: initial; }
+input[name=search] ~ input[type=submit] { display: block !important; }
```

Comment:
Just noticed that the home ext isn't properly sorting HTML headers either, so the css isn't even properly being loaded there. Will fix that too..